microsoft-edge/webdriver-chromium/index.md

```diff
@@ -107,11 +107,19 @@ npm install @microsoft/edge-selenium-tools selenium-webdriver
 
 * * *  
 
+### Using Selenium 4
```

Comment:
It might be more helpful to place this new section above the Selenium 3 section since we state that Selenium 4 is recommended. We should be encouraging people to move to 4 instead of 3.